apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -0,0 +1,46 @@
+{
+  "filterGrades": "Grades",
+  "filterGradesPre": "Pre-reader",
+  "filterGrades25": "Grades 2-5",
+  "filterGrades68": "Grades 6-8",
+  "filterGrades9": "Grades 9+",
+  "filterTeacherExperience": "Educator's experience",
+  "filterTeacherExperienceBeginner": "Beginner",
+  "filterTeacherExperienceComfortable": "Comfortable",
+  "filterStudentExperience": "Student's experience",
+  "filterStudentExperienceBeginner": "Beginner",
+  "filterStudentExperienceComfortable": "Comfortable",
+  "filterPlatform": "Classroom technology",
+  "filterPlatformComputers": "Computers",
+  "filterPlatformAndroid": "Android",
+  "filterPlatformIos": "iPad/iPhone",
+  "filterPlatformNoInternet": "Poor or no internet",
+  "filterPlatformNoComputers": "No computers or devices",
+  "filterTopics": "Topics",
+  "filterTopicsScience": "Science",
+  "filterTopicsMath": "Math",
+  "filterTopicsHistory": "Social Studies",
+  "filterTopicsLa": "Language Arts",
+  "filterTopicsArt": "Art, Media, Music",
+  "filterTopicsCsOnly": "Computer Science only",
+  "filterActivityType": "Activity type",
+  "filterActivityTypeOnlineTutorial": "Self-led tutorials",
+  "filterActivityTypeLessonPlan": "Lesson plan",
+  "filterLength": "Length",
+  "filterLength1Hour": "One hour",
+  "filterLength1HourFollow": "One hour with follow-on",
+  "filterLengthFewHours": "A few hours",
+  "filterProgrammingLanguage": "Language",
+  "filterProgrammingLanguageBlocks": "Blocks",
+  "filterProgrammingLanguageTyping": "Typing",
+  "filterProgrammingLanguageOther": "Other",
+  "filterActivityTypeRobotics": "Robotics",
+  "backButtonBack": "Back to all tutorials",
+  "filterHeaderFilterBy": "Filter By",
+  "filterHeaderShowFilters": "Filters",
+  "filterHeaderHideFilters": "Apply",
+  "roboticsButtonText": "Got robots? <a href='/learn/robotics'>Use these activities</a> and make a tangible Hour of Code for students of any age!",
```

Comment:
Usually I put a param like `<a href='%{robotics_url}'>` and then you can do `I18n.t(:roboticsButtonText, robotics_url: '/learn/robotics')`.